atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbMetricNames.java

```diff
@@ -58,6 +58,8 @@ private CellFilterMetrics() {
     public static final String SWEEP_TS = "sweepTimestamp";
     public static final String LAST_SWEPT_TS = "lastSweptTimestamp";
     public static final String LAG_MILLIS = "millisSinceLastSweptTs";
+    public static final String BATCH_SIZE = "batchSizeHistogram";
```

Comment:
We probably don't need to say it's a histogram; the metrics will have names like `blah.batchSize.p99` that seem to imply histograms